server/plugin/webhook.go

```diff
@@ -350,6 +350,20 @@ func (p *Plugin) excludeConfigOrgMember(user *github.User, subscription *Subscri
 	return p.isUserOrganizationMember(githubClient, user, organization)
 }
 
+func (p *Plugin) includeOnlyConfigOrgMembers(user *github.User, subscription *Subscription) bool {
+	if !subscription.IncludeOnlyOrgMembers() {
+		return false
+	}
+
+	githubClient, err := p.GetGitHubClient(context.Background(), subscription.CreatorID)
+	if err != nil {
+		p.client.Log.Warn("Failed to get user info", "error", err.Error())
+		return false
+	}
+
+	return !p.isUserOrganizationMember(githubClient, user, p.getConfiguration().GitHubOrg)
```

Comment:
Just for readability to have the success state obvious at the bottom of the function

```suggestion
	if !p.isUserOrganizationMember(githubClient, user, p.getConfiguration().GitHubOrg) {
	    return false
	}
	
	return true
```

Reply:
Done 👍 https://github.com/mattermost/mattermost-plugin-github/pull/797/commits/f77d0fa6683d74535f12ff5855e5aeb581465e18